models/sequelize.js

```diff
@@ -0,0 +1,79 @@
+import Sequelize from 'sequelize';
+import bcrypt from 'bcrypt';
+import zlib from 'zlib';
+
+import { DataTypes } from 'sequelize';
+
+const sequelize = new Sequelize('todo', 'postgres', 'postgres', {
+    host: 'localhost',
```

Comment:
Do not show your host and port for security reasons. I know this is localhost but keep them in your environment variables.